controllers/k8ssandra/k8ssandracluster_controller.go

```diff
@@ -325,23 +332,27 @@ func (r *K8ssandraClusterReconciler) reconcile(ctx context.Context, kc *api.K8ss
 		}
 	}
 
-	// Reconcile Stargate across all datacenters
+	kcLogger.Info("All dcs reconciled")
+
+	// Reconcile Stargate and Reaper across all datacenters
 	stargateAuthSchemaReconciled := false
+	reaperBackendSchemaReconciled := false
```

Comment:
What do you think about moving the `reconcileStargateAuthSchema` call and corresponding code in reaper_reconciler.go (lines 142 - 152) here, before the for loop? It would be something like this:

```
if cluster has stargate then
    reconcile stargate auth schema

if cluster has reaper then
    reconcile reaper schema

for each DC
    reconcile stargate
    reconcile reaper
```
While a somewhat minor change, I think it would simply the code a bit. It would eliminate some branching as well as eliminate the `stargateAuthSchemaReconciled` and `reaperBackendSchemaReconciled` vars.

Reply:
Agreed, will do as you suggested.